model/src/cfd.rs

```diff
@@ -3345,15 +3465,15 @@ mod tests {
     #[test]
     fn given_current_settlement_in_12_hours_and_candidate_in_19_then_7_hour_extension() {
         for now in common_time_boundaries() {
-            let current_event_id = BitMexPriceEventId::with_20_digits(now + 12.hours());
-            let candidate_event_id = BitMexPriceEventId::with_20_digits(now + 19.hours());
```

Comment:
Yeah, I am not super happy with `to` and `from` neither, but I found `candidate` super confusing, because for me it indicates that it is only a candidate but might not be used (i.e. something else would be used instead). That's not what we want to express here. It's literally a to-from scenario (from a cost perspective, as in the hours from and to you get charged for). Happy to see a follow up on that that changes it throughout (but please not candidate :)